upload_build.sh

```diff
@@ -53,12 +53,20 @@ contentType="application/x-compressed-tar"
 dateValue=`date -R`
 stringToSign="PUT\n\n${contentType}\n${dateValue}\n${resource}"
 signature=`echo -en ${stringToSign} | openssl sha1 -hmac ${GCLOUD_SECRET} -binary | base64`
+fullUrl="https://storage.googleapis.com${resource}"
+
+# Failsafe - don't overwrite existing uploads!
+if curl --head --fail $fullUrl 2>/dev/null; then
```

Comment:
This means if we run into a case where there is any problem with the binary when building from `mainnet` like on Monday then we *have* to do a version bump in order to get a new binary and bump for any change and no updates to a binary for a release for any reason?